images/monitoring/Makefile

```diff
@@ -21,10 +21,14 @@ build:
 	docker build -t $(IMG):$(TAG) -f Dockerfile ../..
 	docker tag $(IMG):$(TAG) $(IMG):latest
 
+clean_build:
```

Comment:
The usual practice is to have a "clean" rule that removes cached stuff, then you do `make clean && make build`. You can have a shortcut rule "build_clean" that depends on both, of course. But ideally not a duplicate build rule that can get out of sync easily.

Reply:
Thanks for pointing this out. I removed the build_clean rule. 

I was using the build_clean rule for debugging the dependencies. Because I was using `ls` to check the mail dependencies are added to the correct directory. I want it to always print out the directory structure. I don't think the clean rule will be need in the future.